application/helpers/admin/import_helper.php

```diff
@@ -2416,10 +2416,7 @@ function TSVImportSurvey($sFullFilePath)
                     }
                     // Insert sub question and keep the sqid for multi language survey
                     $question = new Question();
-                    //$question->attributes = $insertdata; : this broke call of $this when test rules when save : PLEASE : don't use
-                    foreach($insertdata as $attribute=>$value) {
-                        $question->setAttribute($attribute,$value);
-                    }
+                    $question->setAttributes($insertdata,false); //$question->attributes = $insertdata; : this broke call of $this when test rules when save : PLEASE : don't use
```

Comment:
nice!

Reply:
Maybe, but in general : must move from `$object->attributes = array()` to $object->setAttributes(array(),(true|false));`

I don't search on the file if we have another …

Here : it'sba bug fix, since other are not bug fix : i didn't touch […](https://github.com/LimeSurvey/LimeSurvey/commit/d67f292dfabd747ff5cfc97a3cd35217732084bb#commitcomment-29528163)